openc3/lib/openc3/api/tlm_api.rb

```diff
@@ -161,8 +161,8 @@ def inject_tlm(target_name, packet_name, item_hash = nil, type: :CONVERTED, scop
     # @param args The args must either be a string followed by a value or
     #   three strings followed by a value (see the calling style in the
     #   description).
-    # @param type [Symbol] Telemetry type, :RAW, :CONVERTED (default), :FORMATTED, or :WITH_UNITS
-    def override_tlm(*args, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
+    # @param type [Symbol] Telemetry type, :ALL (default), :RAW, :CONVERTED, :FORMATTED, :WITH_UNITS
+    def override_tlm(*args, type: :ALL, scope: $openc3_scope, token: $openc3_token)
```

Comment:
This is an API change we should note but I think it makes more sense. Especially because PacketViewer shows the WITH_UNITS by default and a call to override_tlm now changes the value in PacketViewer (least surprise).